static/js/list_widget.js

```diff
@@ -10,7 +8,7 @@ const DEFAULTS = {
 // This function describes (programmatically) how to use the ListWidget.
 // ----------------------------------------------------
 
-exports.validate_opts = (opts) => {
+export const validate_opts = (opts) => {
```

Comment:
Oh, this comes up a few times—do we want to convert exported arrow functions to the more common `function` style instead?

```js
export function validate_opts(opts) {
```

Reply:
Probably?  Not super important.